CHANGELOG.asciidoc

```diff
@@ -32,6 +32,13 @@ https://github.com/elastic/beats/compare/v6.2.4...6.2[Check the HEAD diff]
 
 *Affecting all Beats*
 
+- Fix panic when Events containing a float32 value are normalized. {pull}6129[6129]
```

Comment:
It seems to me there are too many entries.

Reply:
Ah, yes, I'll clean it up.